ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -773,6 +773,7 @@
     "AccessControl": [
       {
         "UserRoles": [
+          "[RESPONDENTSOLICITORONE]",
```

Comment:
this stuff isn't correct dharm please revert. 

respondent2ClaimResponseType should only be seen by RESPONDENTSOLICITORTWO. When the user has this role they will be able to do whatever update they need to do in the backend.